online-shopping-website/frontend/src/Components/Profile/AdminPage.js

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import Grid from '@mui/material/Grid';
+import { Link } from "react-router-dom";
+import { DataGrid } from '@mui/x-data-grid';
+import Button from '@mui/material/Button';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Typography } from '@mui/material';
+import axios from 'axios';
+import { ViewOrders } from './ViewOrders';
+import { useCookies } from 'react-cookie';
+import { borderRight } from '@mui/system';
+
+export const AdminPage = () => {
+    //making datagrid colums to see all users
+    const columns = [
+        {
+            field: 'delete',
+            headerName: 'Delete',
+            renderCell: (params) => (
+                <Button className="sellerButton GreenButtonText" variant="text" onClick={() => removeUser(params.id)}>
+                    <DeleteIcon />
+                </Button>
+            ),
+            sortable: false,
+            filterable: false,
+            hideable: false
+        },
+        {
+            field: 'id',
+            headerName: 'User ID',
+        },
+        {
+            field: 'active',
+            headerName: 'Activity',
+        },
```

Comment:
Maybe we should move the "Active" column to the end of the row? I think it's less important than the other data to see first.

Reply:
i'll move the column to the end of the row no prob! I think I know how to change what values are displayed i'll fix that asap ! 😎